lib/src/js/analyze.js

```diff
@@ -43,7 +43,7 @@ function getBehaviors(results, filePath) {
     if (behavior.contentHref != filePath) continue;
 
     behaviors.push({
-      name: behavior.is,
+      name: behavior.is.replace('Polymer.', ''),
```

Comment:
Oh, this is what is building the json object which I am modifying later on based on the class names you provide.